### ISSUE TITLE
MvvmCross support for Windows 10 apps(UWP)

### ISSUE THREAD

**Issue by Suthahar** [MvvmCross/MvvmCross#1066]
I have created Windows 8.1 app with mvvmcross .Now I have migrated same app into windows 10 (UWP) app .I am not get any mvvmcross reference issue . While running application I am getting splash screen after some min I am getting below error 

An exception of type 'Cirrious.CrossCore.Exceptions.MvxException' occurred in Cirrious.CrossCore.dll but was not handled in user code

Additional information: Could not find plugin type in assembly.

I have done step by step debug solution the Setup.Initialize() method is not calling while app launch

can you please let me know .Is it mvvmcross library is compatible with windows 10 apps(UWP).

if its compatible please give best solution

**Comment by Cheesebaron** [comment]
Do you have a bootstrap folder with the bootstrap classes needed for the plugins you are using?

**Comment by Suthahar** [comment]
Hi cheesebaron,

Yes I have bootstrap folder for SQLite,Messenger plugin

**Comment by CuriousCurmudgeon** [comment]
This sounds like a SQLite issue with Windows 10. Here is one potential solution: http://ooiks.com/blog/how-to-2/sqlite-problem-in-universal-windows-platform. Googling for "sqlite Windows 10 UWP" will get you quite a few relevant hits.

**Comment by martijn00** [comment]
Fixes for nuspecs for uwp have been made. Discussion on further support can be done in #1009

**Comment by NPadrutt** [comment]
I installed the v4 beta 2 in a solution with an Android and a UWP project. In the Android project it works so far, but in the windows project there is no symbol for the setup found (I tried MvxWindowsSetup and MvxStoreSetup). It seems like an issue with the references. But when I try to add it manually it states that the reference already exists. Do I have to do something different for a Windows project?
